### DISCUSSION TITLE
Is the `style()` function's `debugId` parameter safe to use?

### DISCUSSION THREAD

**Discussion by maranomynet** [vanilla-extract-css/vanilla-extract#1200] [Q&A] [answered]
Hi I notice that the `style()` function accepts an optional second parameter called `debugId`, which allows me to manually construct the debug id segment of the generated class-name.

This is very nice, and helps me in cases where I'm writing higher-order style generators. (I.e. running `style()` in utility helpers outside the `*.css.ts` file.)

However, I don't see any mention of this parameter in the official documentation.

Is this a mistake in the docs, or am I using an unsafe API that's intended to be private?

**Comment by askoufis** [comment, marked as answer]
I could've sworn this was documented somewhere, but I guess it isn't. This parameter is safe to use, and is not private API. A PR to the docs would be appreciated if you feel up to it.
